### PR TITLE
Add initial version of transaction overview slides

### PR DESCRIPTION
**Pull request by dariothuerkauf**
# Description

* Created the first version of the slides on transactions
* Added 5 new figures (figure on each of the 4 different transaction types and transaction-process)
* Known issue: Slide regarding UTXO's and unlocking condition yet missing, placeholder instead
* Added tikzlibraries (decorations.pathreplacing, decorations.markings, positioning) at the end of Config file

## Type of Change

- [x] Major feature (New files or assets, large changes to files)

# Checklist:

- [x] I have performed a self-review of my own changes
- [x] I have made sure that any changed LaTeX files compile properly
- [x] I have assigned at least one reviewer

### CODE REVIEW

File: slides/transactions.tex [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  	\item<2->{Must contain at least one in- and output}
+  	\item<3->{Sum of inputs must be at least as high as sum of outputs}
+	\item<4->{Credit can be divided to any number of output}
+\end{itemize}
+\end{frame}	
+%%%
+
+
+%%%
+\begin{frame}{Unlocking Condition \& UTXO model}
```
Comment (Chainomics) [inline slides/transactions.tex:43]: Empty slide?

File: slides/transactions.tex [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+\begin{frame}{Structure of a Transaction}
+	\centering
+	\begin{figure}
+	\input{../assets/figures/transaction-structure.tex}
+	\end{figure} 
+\vspace{1em}
+\begin{itemize}
+  	\item<2->{Must contain at least one in- and output}
+  	\item<3->{Sum of inputs must be at least as high as sum of outputs}
+	\item<4->{Credit can be divided to any number of output}
```
Comment (Chainomics) [inline slides/transactions.tex:36]:
```suggestion
```

**Review by Chainomics — CHANGES_REQUESTED** [review]
Unfortunately, there is an empty slide, i.e. unlocking condition and UTXO model. Please make sure that the slide deck is complete before creating a pull request.

File: slides/transactions.tex (hunk 1 above)
Comment (dariothuerkauf) [reply slides/transactions.tex]: Updated in commit 3b98723b34ab62fca4cbb8066335dd4f2d9224fa

File: slides/transactions.tex (hunk 2 above)
Comment (dariothuerkauf) [reply slides/transactions.tex]: Changed in commit 3b98723b34ab62fca4cbb8066335dd4f2d9224fa

**Review by Chainomics — APPROVED** [review]

**Pull request merged**